lib/pipeline_controller.py

```diff
@@ -36,6 +60,14 @@ def __init__(self):
 
     def run(self):
         """Main method for the class -- runs the pipeline"""
+        self.logger.info("Getting regular branch hours from Redshift")
+        location_hours_dict = self.get_location_hours_dict()
+        self.shoppertrak_api_client = ShopperTrakApiClient(
```

Comment:
What is the advantage to moving this out of the constructor? Feels like a more natural fit to me.

Reply:
I moved it out because the location_hours_dict has to be set now. I just changed it so that the client is created in the constructor with an empty location_hours_dict which is then set down below.